addons/purchase_stock/tests/test_purchase_lead_time.py

```diff
@@ -33,7 +33,7 @@ def test_00_product_company_level_delays(self):
         self.assertEqual(purchase.date_order, order_date, 'Order date should be equal to: Date of the procurement order - Purchase Lead Time - Delivery Lead Time.')
 
         # Check scheduled date of purchase order
-        schedule_date = datetime.combine(order_date + timedelta(days=self.product_1.seller_ids.delay), time.max).replace(microsecond=0, second=0)
+        schedule_date = datetime.combine(order_date + timedelta(days=self.product_1.seller_ids.delay), time.min).replace(tzinfo=None, hour=12)
```

Comment:
you could use time(12) instead of time.min.replace(hour=12)

Reply:
We could use the company timezone to avoid this hack and also resolve this issue for company with UTC+13 +14